master/introduction/index.md

```diff
@@ -21,8 +21,29 @@ you can achieve fine-grained control over communications
 between containers, virtual machine workloads, and 
 bare metal host endpoints.
 
-Proven in production at scale, {{site.prodname}} features 
-integrations with Kubernetes, OpenShift, Docker, 
-Mesos, DC/OS, and OpenStack.
+Proven in production at scale, {{site.prodname}} v3.0 features 
+integrations with Kubernetes, OpenShift, and OpenStack.
 
-<a href="/{{page.version}}/getting-started/" class="btn btn-primary btn-lg">Get started</a>
\ No newline at end of file
+> **Note**: For integrations with the  Mesos, DC/OS, and Docker 
+> orchestrators, use [Calico v2.6](/v2.6/introduction/). We plan 
+> to resume support for these orchestrators in a future 
+> v3.x release.
+{: .alert .alert-info}
+
+# Get started
+
+<a href="/{{page.version}}/getting-started/kubernetes/" class="btn btn-primary btn-sm">Kubernetes</a>
+<a href="/{{page.version}}/getting-started/openshift/installation" class="btn btn-primary btn-sm">OpenShift</a>
+<a href="/{{page.version}}/getting-started/openstack/" class="btn btn-primary btn-sm">OpenStack</a>
+
+# How it works
+
+![Calico overview diagram]({{site.baseurl}}/images/calico-arch-gen.svg){: width="60%" }
+
+- **calicoctl**: simple command-line management interface.
+
+- **orchestrator plugins**: provide close integrations with a variety of popular orchestrators
+
+- **key/value store**: storage for Calico's workload, policy, and configuration state
```

Comment:
For symmetry, should probably be "_stores_ Calico's workload, policy, and configuration state."